src/main/java/io/cdap/plugin/splunk/common/config/BaseSplunkConfig.java

```diff
@@ -152,6 +181,14 @@ public Integer getMaxRetryJitterWait() {
 
   public void validate(FailureCollector collector) {
     IdUtils.validateReferenceName(referenceName, collector);
+    try {
+      if (!containsMacro(PROPERTY_URL)) {
+        getUrl();
+      }
+    } catch (IllegalArgumentException e) {
+      collector.addFailure(e.getMessage(), null)
```

Comment:
validation error message should contain field name

Reply:
Done.